doc/source/hello-world.rst

```diff
@@ -1,25 +1,25 @@
 Installing and Hello World
 ==============================
 
-Release versions are posted on the GitHub `releases page <https://github.com/epigen/pypiper/releases>`_. You can install the latest version directly from GitHub using pip:
+Release versions are posted on the GitHub `releases page <https://github.com/epigen/pypiper/releases>`_. You can also install the latest version directly from pypi (under the name ``piper``) using pip:
```

Comment:
This was the resolution to the name conflict?

Reply:
yeah, for now.